libraryapi/pergamum.py

```diff
@@ -42,16 +43,17 @@ def __init__(self, base_url: str) -> None:
                 f"{base_url}/web_service/servidor_ws.php?wsdl",
                 transport=Transport(session=session),
             )
-        except HTTPError as e:
+        except HTTPError as error:
             raise PergamumWebServiceException(
-                message=f"{base_url}/web_service/servidor_ws.php?wsdl returned {e.response.status_code}"
-            )
+                message=f"{base_url}/web_service/servidor_ws.php?wsdl returned {error.response.status_code}"
+            ) from error
```

Comment:
Não faz sentido fazer o chaining dessa exceção porque não temos interesse na stack trace , não é por isso que estamos subindo a excecao

Reply:
hum, na real nem entendo muito pra que isso server (from ...), foi adicionado só para o pylint não ficar emitindo "Consider explicitly re-raising using the 'from' keyword". Tirei então.